tests/if_transform/secret_condition_with_secret_tensor.mlir

```diff
@@ -0,0 +1,27 @@
+module {
```

Comment:
Placeholder comment to remind me about FileCheck tests

Reply:
Also: note to add tests with if statements that do not yield any values, including ones that do not have an else branch

Reply:
PS: you should be able to add all tests into a single file